server/routes/events.js

```diff
@@ -50,17 +50,15 @@ router.patch("/updateEvent/:id", async (req, res, next) => {
 });
 
 // Get event by id and update
-router.get("/getEvent/:id", async (req, res, next) => {
+router.get("/get/:id", async (req, res, next) => {
```

Comment:
Since we are doing a GET request we don't need to specify get in the route 
Instead, you can just do:
`router.get("/:id", async (req, res, next) => {`